{"name": "test_send_push_page_data_messages", "status": "failed", "statusDetails": {"message": "AssertionError: True\nassert 'Доставлено' in 'Отложено'", "trace": "browser = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"0ad5552881319cc80d6785ef243269d4\")>\n\n    def test_send_push_page_data_messages(browser):\n        login_page = Action_Login(browser)\n        login_page.go_to_site()\n        login_page.autorization()\n        main_page = Action_Main(browser)\n        main_page.open_create_message()\n        message_create_page = Action_MessageCreate(browser)\n        message_create_page.enter_data_message_schema('заголовок', 'сообщение', '79101234567', 'push')\n        message_create_page = Action_MessageCreateStatus(browser)\n        message_create_page.open_data_messages()\n        data_messages_page = Check_DataMessages(browser)\n        check = data_messages_page.check_message_status()\n>       assert 'Доставлено' in check, browser.get_screenshot_as_file(test_send_push_page_data_messages.__name__ + ' ' + 'FAILED.png')\nE       AssertionError: True\nE       assert 'Доставлено' in 'Отложено'\n\nC:\\Users\\User\\PycharmProjects\\testismg\\Tests\\test_send_message.py:67: AssertionError"}, "start": 1652779756004, "stop": 1652779761273, "uuid": "43229c75-9065-448b-91c9-02dd0784336d", "historyId": "d1921cfde3c1aad8d3823df5efc84669", "testCaseId": "cec5ffb32bb3f0b041816e032fb6cea9", "fullName": "Tests.test_send_message#test_send_push_page_data_messages", "labels": [{"name": "parentSuite", "value": "Tests"}, {"name": "suite", "value": "test_send_message"}, {"name": "host", "value": "DESKTOP-13RCMAF"}, {"name": "thread", "value": "17840-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Tests.test_send_message"}]}